### PR TITLE
Add utilities for converting from/to rr graph inodes and routing data…

### PR DESCRIPTION
**Pull request by litghost**
…base

This is useful for diagnosing router behavior.

The amount of time to write the node_map via pickle is very small (~100 msec).

### CODE REVIEW

File: xc7/utils/prjxray_lookup_inode.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
+""" Convert rrgraph inode back into graph_node_pkey and print some information.
```
Comment (mithro) [inline xc7/utils/prjxray_lookup_inode.py]:
```python
#!/usr/bin/env python3
```

File: xc7/utils/prjxray_find_inode.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
+""" Find an rr node by specifying a wire name (e.g. INT_R_X3Y149/WL1BEG0).
```
Comment (mithro) [inline xc7/utils/prjxray_find_inode.py]:
```python
#!/usr/bin/env python3
```

File: xc7/utils/prjxray_routing_import.py [hunk 3]
```diff
@@ -694,6 +700,11 @@ def main():
                 import_graph_edges(conn, graph, node_mapping)
             )
 
+        print('{} Writing node map.'.format(now()))
+        with open(args.write_rr_node_map, 'wb') as f:
+            pickle.dump(node_mapping, f)
```
Comment (mithro) [inline xc7/utils/prjxray_routing_import.py]: Is using a pickle verse a json file more performant?

Comment (litghost) [reply xc7/utils/prjxray_routing_import.py]: Pretty much.  This is a diagnostic leaf, I wanted a low impact both in runtime and disk space.

**Pull request merged**